### PR TITLE
Dashboard contextual & accessibility changes

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/pages/supplier/dashboard/template.njk [hunk 1]
```diff
@@ -2,39 +2,82 @@
...
+      </div>
+
+      <div class="nhsuk-u-margin-bottom-5" data-test-id="dashboard-submit-for-moderation-button">
+        {{ button({
+          "text": "Submit for moderation",
+          "href": submitForModerationUrl
+        }) }}
+      </div>
+
+      <div class="nhsuk-u-padding-4 nhsuk-u-margin-bottom-5" style="border: 5px solid #005eb8; background: #fff">
```
Comment (kat1906) [inline app/pages/supplier/dashboard/template.njk]: please extract styles into classes

File: app/pages/supplier/dashboard/template.njk [hunk 2]
```diff
@@ -2,39 +2,82 @@
...
-      {{ button({
-        "text": "Preview Marketing page",
-        "classes": "nhsuk-button--secondary",
-        "href": previewUrl
+      {{ tag({
+        text: 'Supplier name',
+        classes: 'nhsuk-u-margin-bottom-4 bc-c-tag-outline'
       }) }}
+
+      <h1>&lt;Catalogue Solution name&gt; listing page – all sections</h1>
```
Comment (kat1906) [inline app/pages/supplier/dashboard/template.njk]: do you want the < and > and hardcoded solution name? looks a bit weird

File: app/pages/supplier/dashboard/template.njk [hunk 3]
```diff
@@ -2,39 +2,82 @@
...
+
+      <h2 class="nhsuk-u-margin-bottom-5">Explain what your Catalogue Solution is for and how it works. This information will be published on the Buying Catalogue to help buyers understand if it meets their needs.</h2>
```
Comment (kat1906) [inline app/pages/supplier/dashboard/template.njk]: can we extract the text into a manifest.json?

File: app/pages/supplier/dashboard/template.njk (hunk 2 above)
Comment (BenBJSS) [reply app/pages/supplier/dashboard/template.njk]: Was just to match the designs, I can't pull through the solution name yet so I thought it looked more obvious it needed to be sorted with the <>

Comment (kat1906) [reply app/pages/supplier/dashboard/template.njk]: the solution name comes back from the API ```{
  id: '100000-001',
  name: 'Write on Time',
  sections: {...}
}```
so you can get it from there? Just need to extract it from the data that comes in in the controller file

**Pull request merged**